contracts/RewardPool/modules/DelegationRewards.sol

```diff
@@ -403,6 +409,13 @@ abstract contract DelegationRewards is RewardPoolBase, Vesting, RewardsWithdrawa
 
     // _______________ Private functions _______________
 
+    function _changeMinDelegation(uint256 newMinDelegation) private {
+        if (newMinDelegation < 1 ether) { 
+            revert InvalidMinDelegation(newMinDelegation);
+        }
```

Comment:
Please, leave an empty line after `}`, (keep the same styling convention)

Reply:
got it